### PR TITLE
Modification of the log file display

### PR DESCRIPTION
**Pull request by lyt700**
Modified the current log file display method based on master branch version.
Log file will directly displayed on GUI and automatically update when new gesture/ gesture sequence is logged into the file.

### CODE REVIEW

File: guiManager.py [hunk 1]
```diff
@@ -521,24 +536,25 @@ def add_new_command(self):
...
-            pdf.cell(200, 10, txt=line, ln=count, align="Left")
-            count +=1
-        file.close()
-        pdf.output("logfile.pdf")
-        subprocess.call(["open", "logfile.pdf"])
-
-    def delete_log_file(self):
-        if os.path.exists("logfile.pdf"):
-            os.remove("logfile.pdf")
+    ''' The code block below is old version of log file display, we can remove it later'''
```
Comment (lgint8042) [inline guiManager.py]: I think we can probably just go ahead and delete this since it won't be used anymore.

File: guiManager.py [hunk 2]
```diff
@@ -443,9 +447,20 @@ def __init__(self, name, window, cap, on_ear_change, initial_ear, on_low_contras
                 add_button = Button(small_frame, text="Add New Command", command=self.add_new_command)
                 add_button.grid(row=self.row_index, column=self.command_index + 1, pady=10)
 
-            elif element["format"] == "button":
-                self.log_button = Button(self, text = 'Click to see contents of the logfile', command = self.open_log_file).pack()
-                # self.delete_log_file()
+            elif element["format"] == "logfile":
+                self.is_logfile = True
+                self.scro = Scrollbar(self)
```
Comment (lgint8042) [inline guiManager.py]: Can this be changed to `self.scroll` or `self.scrollBar`?

File: guiManager.py (hunk 1 above)
Comment (lyt700) [reply guiManager.py]: Would be delete those code before merge when the request is accept

File: guiManager.py (hunk 2 above)
Comment (lyt700) [reply guiManager.py]: changed

File: guiManager.py (hunk 1 above)
Comment (lyt700) [reply guiManager.py]: deleted

File: guiManager.py (hunk 2 above)
Comment (jcf1991) [reply guiManager.py]: self.scroll_bar would be more appropriate by pythonic conventions, methinks. I agree that it should change. I'll make a PR that changes it if you don't beat me to it!

**Review by jcf1991 — APPROVED** [review]
I agree (more or less) with Landan's comments regarding the scrollbar name and the comment block, but I don't think those should prevent us from merging this. Let's merge it, and write more code that addresses these issues of convention. 

I also think the order the log displays in should be different reversed (as per prior instructions from Iqbal in a team meeting), but I addressed that issue in another PR.

**Review by codieorberson — APPROVED** [review]

**Review by Danielle333 — APPROVED** [review]

**Pull request merged**